lua/metals.lua

```diff
@@ -1,9 +1,15 @@
 local lsp = vim.lsp
 local vim = vim
+local api = vim.api
 local util = require'metals.util'
 
 local M = {}
 
+M.decoration = {
```

Comment:
I don't love that this exposes `namespace` to the outside world when it probably shouldn't be. Another option could be to instead set something like:

```vim
let g:metals_decoration_color = 'Conceal'
```

Then in `plugin/metals.vim` do a check for it or assign it. Something like is done here: https://github.com/haorenW1025/completion-nvim/blob/2f579083a3585dffbf799ccb8bd5d89b175ca761/plugin/completion.vim#L8-L10

That way we wouldn't expose this entire thing. What do you think?